superai/apis/meta_ai/project_ai.py

```diff
@@ -42,7 +40,7 @@ def get_models(self, app_id: str, assignment: str = None, active=None):
         models = app_assignments.model
         models.id()
         models.name()
-        data = sess.perform_op(op)
+        data = self.ai_session.perform_op(op, app_id=app_id)
```

Comment:
Where is the `self.ai_session` defined?

Reply:
Thats coming from the `AiApiBase`, which this is inherting from.